testgrid/tgapi/pkg/testrun/testrun.go

```diff
@@ -65,7 +65,24 @@ func Create(id string) error {
 func List(limit int, offset int, searchRef string) ([]types.TestRun, error) {
 	pg := persistence.MustGetPGSession()
 
-	query := `select ref, created_at from testrun where lower(ref) like '%' || $1 || '%' order by created_at desc`
+	query := `
+select 
+	ref,
+	created_at,
+	COUNT(id) as total,
+	COUNT(id) FILTER (WHERE is_success AND (NOT is_unsupported OR is_unsupported IS NULL)) AS successes,
+	COUNT(id) FILTER (WHERE NOT is_success AND (NOT is_unsupported OR is_unsupported IS NULL) AND finished_at IS NOT NULL) as failures,
+	MAX(finished_at) as latest_completion,
+	COUNT(id) FILTER (WHERE finished_at IS NULL) as pending_runs
+from testrun 
+	left join testinstance on testrun.ref=testinstance.testrun_ref
+where
+	lower(ref) like '%' || $1 || '%'
```

Comment:
can you change this to `or`

Reply:
I really didn't want to touch that since it's from the previous query